inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/src/matchers/base_matcher.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "matchers/base_matcher.hpp"
+#include "common_test_utils/common_utils.hpp"
+
+
+SubgraphsDumper::iMatcherConfig::Ptr SubgraphsDumper::Matcher::get_config(const std::shared_ptr<ngraph::Node> &node) {
+    for (const auto &cfg : default_configs) {
+        if (cfg->op_in_config(node)) {
```

Comment:
Can be united in one cycle

Reply:
Not really, it will break configuration searching logic. The algorithm is following: searching for config for certain OP -> if it's not found -> search for fallback config -> if it's not found create default config. Uniting  in the one loop will lead to undefined behaviour - either fallback config will be returned or OP config depending on which appear first.